src/pages/Widgets.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import "./Widget.css";
+
+import Todo from "../Component/Todo/Todo";
+import Greeting from "../Component/Greeting/Greeting";
+import Time from "../Component/Time/Time";
+import Weather from "../Component/Weather/Weather";
+import Quotes from "../Component/Quotes/Quotes";
+
```

Comment:
make a separate file index.js and import all these components from there, read about barrel exports